summarizer/summarize.test.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const summarize = require('./summarize');
+
+describe('summarize tests...', () => {
+  const text = fs.readFileSync('./summarizer/text.txt', 'utf-8');
```

Comment:
This should go in a `before` test setup call.  You can also drop the `readFileSync` and just use `readFile`, since we can use async calls in a test setup function.